packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -406,6 +410,13 @@ class CupertinoDatePicker extends StatefulWidget {
   /// Whether to to show day of week alongside day. Defaults to false.
   final bool showDayOfWeek;
 
+  /// The uniform height of all children.
```

Comment:
The date picker doesn't have any children that can be specified (e.g. like a `Column` does).  Can you clarify which children you're referring to, or maybe use more specific terminology for the children (buttons, date fields, calendar day grid items, etc.).